src/models/data.py

```diff
@@ -169,6 +177,10 @@ def ds_folder_to_np(self,
         # first, x
         x_np = x_np.reshape(x_np.shape[0], x_np.shape[1], x_np.shape[2] * x_np.shape[3])
         x_np = np.moveaxis(np.moveaxis(x_np, 0, 1), -1, 0)
+        # then, the x month
+        assert len(y.time) == 1, f'Expected y to only have 1 timestamp! Got {len(y.time)}'
+        target_month = datetime.strptime(str(y.time.values[0])[:-3], '%Y-%m-%dT%H:%M:%S.%f').month
+        x_months = np.array([target_month] * x_np.shape[0])
```

Comment:
doesn't this just create a vector of the `target_month` rather than the months of the input timesteps ? or is this what we want? 

I.e. to know that we are looking at predicting June vs. to know that we are using March, April, May to predict June? Or is this something implicitly picked up?

Reply:
Since everything is flattened, it wouldn't really make sense to input the prediction months (you'd end up with a list of integers); I thought this would be better captured just by adding the prediction month.

(So yes; I am expecting the input months to be implicitly picked up). We can change this if it doesn't seem to work